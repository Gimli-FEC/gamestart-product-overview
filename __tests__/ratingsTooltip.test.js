import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';

import Tooltip from '../client/components/ratingsTooltip.jsx';

const propsObj = {
  message: `
      Rating: 4 out of 5\n
      (based on 100 reviews)\n
      5⭑: 20% \n
      4⭑: 20% \n
      3⭑: 20% \n
      2⭑: 20% \n
      1⭑: 20%`,
};

afterEach(cleanup);

test('renders', () => {
  const { asFragment } = render(<Tooltip message={propsObj.message} />);
  expect(asFragment()).toMatchSnapshot();
});
