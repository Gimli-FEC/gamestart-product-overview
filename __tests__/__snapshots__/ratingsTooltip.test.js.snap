// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <span
    class="ratingsTooltip__Popup-sc-1d6glbo-0 irRflo"
  >
    <span
      class="ratingsTooltip__Trigger-sc-1d6glbo-1 bevQCd"
      data-testid="tt"
    />
  </span>
</DocumentFragment>
`;
